Upated Babyshark VTOL config and vtol_defaults

sfuhrer opened the pull request:
Updated Babyshark VTOL config and vtol_defaults

Updated the babyshark default parameters for improved flight performance, as well as two MPC parameters in vtol_defaults for smoother hovering with VTOLS (as advised by @bresch).

test flight:
https://review.px4.io/plot_app?log=d45d7862-adeb-471c-9972-af6121f8d273

FYI @RomanBapst 

@dagar what do you think of having SYS_FORCE_F7DC  = 2 in the config (or generally set to this value for all VTOLS)? When using avoidance we needed it, had a crash on an other VTOL before we set it.

dagar commented:
> @dagar what do you think of having SYS_FORCE_F7DC = 2 in the config (or generally set to this value for all VTOLS)? When using avoidance we needed it, had a crash on an other VTOL before we set it.

Does the Babyshark ship with this flight controller? Generally we should try and make the airframes map exactly to what a regular user would have, but in this case setting `SYS_FORCE_F7DC` is harmless for the other flight controllers.

I'd also like to emphasize the importance of coming back and fixing the core problem when we have a chance (https://github.com/PX4/Firmware/issues/12216).

dagar commented:
The CI failure is because fmu-v2 is out of flash. One quick thing you could do is exclude it from fmu-v2 like this. https://github.com/PX4/Firmware/blob/d139bc5a7c7394038260cb553a3eb12040c9be0c/ROMFS/px4fmu_common/init.d/airframes/13004_quad%2B_tailsitter#L19

sfuhrer commented:
@dagar  okay, addressed your comments. "@board px4_fmu-v2 exclude" only  is for the CI, or what? 

The Babyshark is shipped with a Cube, we switched to Pixhawk4 to have more RAM. 

I've now also added SENS_BOARD_ROT in the config (it's yawed 180 how they ship it)

dagar commented:
> "@board px4_fmu-v2 exclude" only is for the CI, or what?

It prevents the airframe from being included in the px4_fmu-v2 build (the original pixhawk with only 1 MB of flash).
